product_docs/docs/migration_portal/4/03_mp_using_portal/02_mp_overview_project.mdx

```diff
@@ -15,15 +15,15 @@ The Migration Portal Projects page provides detailed information about your migr
 
 Use the following resources to gather information about your migration projects:
 
--  **Compatible**: The **Project Compatibility** gauge displays the color on the basis of the compatibility percentage of the assessed schema.
+-  **Project Compatibility**: The **Project Compatibility** gauge displays the color based on the compatibility percentage of the assessed schema.
```

Comment:
Please check this. Based on the change that was made to the text, I assume my change is correct. 

I revised the rest of the text to be more concise and idiomatic, but I can't gather from this description how the control actually works. Does it get darker or lighter depending on how compatible it is?

Reply:
This correction looks good to me. I checked the UI again, and the element we are referring to here does not seem to have an "official" name. The control indeed changes color depending on how compatible the project is: 

![Screenshot 2024-05-24 at 09 10 46](https://github.com/EnterpriseDB/docs/assets/98893586/dd208e1f-5f9d-4740-a80a-94ead6246883)

As a non-native, I had to google "gauge" to understand what was meant here. Maybe using an adjective to further describe the element could make it easier for people like me to immediately recognize what is meant here? For example: 

```
The **Project Compatibility** circular gauge
```